srcpkgs/uget-integrator/template

```diff
@@ -0,0 +1,21 @@
+# Template file for 'uget-integrator'
+pkgname=uget-integrator
+version=1.0.0
+revision=1
+depends="python3 uget"
+short_desc="Native messaging script to allow uGet to integrate with a browser"
+maintainer="RunningDroid <runningdroid@zoho.com>"
+license="GPL-3.0-or-later"
+homepage="https://github.com/ugetdm/uget-integrator"
+distfiles="https://github.com/ugetdm/uget-integrator/archive/v${version}.tar.gz"
+checksum=bbc85c32d94e2b6a21977c559f6e49cea9613028713df89e91327a23fef19fa9
+
+do_install() {
+	vbin bin/uget-integrator
+
+	# setup chromium integration
+	vinstall conf/com.ugetdm.chrome.json 0644 etc/chromium/native-messaging-hosts/
```

Comment:
Maybe also copy to `etc/opt/chrome/native-messaging-hosts/` so it also works in google-chrome

Reply:
Sorry, I forgot google-chrome was in the repos. I've added a line that `vinstall`s `com.ugetdm.chrome.json` to `etc/opt/chrome/native-messaging-hosts/`.